src/main/java/cz/czechitas/automation/SeleniumActionFacade.java

```diff
@@ -19,40 +19,40 @@ final class SeleniumActionFacade {
 
     private final Random random = new Random();
 
-    final PublicMenuAction horniMenu;
-    final InternalMenuAction interniMenu;
-    final LoginAction prihlasovani;
-    final OrderAction sekceObjednavky;
-    final ApplicationAction sekcePrihlasky;
-    final ApplicationDetail detailPrihlasky;
-    final ProfileAction profil;
+    final PublicMenuAction headerMenu;
+    final InternalMenuAction internalMenu;
+    final LoginAction loginAction;
+    final OrderAction orderAction;
+    final ApplicationAction applicationAction;
```

Comment:
I am thinking about usability.

Wouldn't it be better to name this field as applicationSection? From my point of view, the applicationSection is more intuitive.

Reply:
Resolved